relay-server/src/services/processor.rs

```diff
@@ -1874,6 +1890,8 @@ impl EnvelopeProcessorService {
         } = message;
 
         let received = relay_common::time::instant_to_date_time(start_time);
+        let received_timestamp = UnixTimestamp::from_secs(received.timestamp() as u64);
```

Comment:
```suggestion
        let received_timestamp = UnixTimestamp::from_instant(start_time);
```

There is another usage I cannot comment on (thanks github)